addons/hr_recruitment/wizard/applicant_refuse_reason.py

```diff
@@ -1,14 +1,105 @@
 # -*- coding: utf-8 -*-
 
-from odoo import api, fields, models
+from odoo import api, fields, models, _
+from odoo.tools.misc import get_lang
 
 
 class ApplicantGetRefuseReason(models.TransientModel):
     _name = 'applicant.get.refuse.reason'
+    _inherits = {'mail.compose.message': 'composer_id'}
```

Comment:
Don't do that, define subject/body fields like in /enterprise/hr_appraisal_survey/wizard/appraisal_ask_feedback.py

And _render the template accordingly.

Otherwise, this is a performance killer.

Thanks.

Reply:
I think we should think of a small utility class for wizard sending emails (appraisal, survey, elearning). Could be great as it seems lot of people are copy-pasting code a bit randomly and sometimes take it from the wrong source modules 8-) .

Anyway agree with @tivisse : currently please be as minimal as possible (define subject/body + rendering in a compute), someday we will probably have a tool for those wizards.